node/src/main/scala/org/bitcoins/node/networking/P2PClient.scala

```diff
@@ -112,7 +111,7 @@ case class P2PClientActor(
           handleEvent(message, peerConnection, unalignedBytes)
         context.become(awaitNetworkRequest(peerConnection, newUnalignedBytes))
       case P2PClient.CloseCommand =>
-        logger.info(s"disconnecting from peer $peer")
+        logger.debug(s"disconnecting from peer $peer")
```

Comment:
Why did all these logs get moved from `info` -> `debug`? It seems like these are worthy logs to have at the `info` level. Same with logs in `def reconnecting`

Reply:
They were fine earlier when we connect to just 1 peer but with many connections, most of which are temporary just to determine service, that would lead to way too many log messages.